src/SprykerEco/Yves/Computop/Mapper/Init/PrePlace/PayPalExpressToQuoteMapper.php

```diff
@@ -0,0 +1,105 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Yves\Computop\Mapper\Init\PrePlace;
+
+use Generated\Shared\Transfer\AddressTransfer;
+use Generated\Shared\Transfer\ComputopPayPalExpressInitResponseTransfer;
+use Generated\Shared\Transfer\CountryTransfer;
+use Generated\Shared\Transfer\CustomerTransfer;
+use Generated\Shared\Transfer\QuoteTransfer;
+
+class PayPalExpressToQuoteMapper implements PayPalExpressToQuoteMapperInterface
+{
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     * @param \Generated\Shared\Transfer\ComputopPayPalExpressInitResponseTransfer $computopPayPalExpressInitResponseTransfer
+     *
+     * @return \Generated\Shared\Transfer\QuoteTransfer
+     */
+    public function mapAddressTransfer(
```

Comment:
Ideally this method should be called `mapComputopPayPalExpressInitResponseTransferToAddressTransfer`, it should have `AddressTransfer` as a second parameter and return `AddressTransfer`.